TypeDBConsole.java

```diff
@@ -409,6 +409,17 @@ private boolean runInlineCommandMode(CLIOptions options, List<String> inlineComm
                     } else if (command.isUserCreate()) {
                         boolean success = runUserCreate(driver, isCloud, command.asUserCreate().user(), command.asUserCreate().password());
                         if (!success) return false;
+                    } else if (command.isUserPasswordUpdate()) {
+                        REPLCommand.User.PasswordUpdate userPasswordUpdate = command.asUserPasswordUpdate();
+                        boolean passwordUpdateSuccessful = runUserPasswordUpdate(driver,
+                                isCloud,
+                                options.username,
+                                userPasswordUpdate.passwordOld(),
+                                userPasswordUpdate.passwordNew());
+                        if (passwordUpdateSuccessful) {
+                            printer.info("Please login again with your updated password.");
+                            break;
+                        }
```

Comment:
Do you need an `else { return false; }` ?

Reply:
Good catch! Fixed.